main.go

```diff
@@ -115,10 +115,15 @@ func getPacketConfig(providerConfig string) (packet.Config, error) {
 	if config.LoadBalancerConfigMap == "" {
 		config.LoadBalancerConfigMap = defaultLoadBalancerConfigMap
 	}
+
 	if config.LoadBalancerConfigMap == "disabled" {
 		config.LoadBalancerConfigMap = ""
 	}
 
+	if loadBalancerConfigMap == "none" {
```

Comment:
This will be removed and we'll actually pass "disabled" in as the Configmap name and not "none", which will be parsed in the `loadbalancer.go`